test/check-application

```diff
@@ -135,7 +130,6 @@ class TestApplication(testlib.MachineCase):
 
         # delete image busybox that hasn't been used
         b.wait_present('#containers-images tr:contains("busybox:latest")')
-        b.click('#containers-images tbody tr:contains("busybox:latest") td.listing-ct-toggle')
```

Comment:
Seems weird (or at least unrelated) to drop this line now? Isn't the delete button inside the details? If not, this should become a separate commit.

Reply:
It's weird but the test was already opening that row much more above, https://github.com/cockpit-project/cockpit-podman/pull/68/files/ab7e4acf1ab8c3eac08bdb1ccaba3d74108ac168#diff-c23b7eb2c7c51d1e825ac452faf7eaa1R39

The only reason we needed to open it again here is because we reloaded the page in the past.

I agree that this test needs refactoring to be more readable.